modules/parrableIdSystem.js

```diff
@@ -242,12 +263,16 @@ function fetchId(configParams, gdprConsentData) {
               if (responseObj.ibaOptout === true) {
                 newParrableId.ibaOptout = true;
               }
+              if (responseObj.tpcSupport !== undefined) {
+                newParams.tpcSupport = responseObj.tpcSupport;
+                newParams.tpcUntil = Math.floor((Date.now() + responseObj.tpcSupportTtl) / 1000);
```

Comment:
Will the endpoint be sending tpcSupportTtl in seconds or milliseconds? I don't recall what the NEST story says.

Reply:
Can this computation be written as a helper function like `newParams.tpcUntil = epochFromTtl(responseObj.tpcSupportTtl)` or something like it.